src/BlazorWebView/src/Maui/build/Microsoft.AspNetCore.Components.WebView.Maui.targets

```diff
@@ -44,6 +44,14 @@
       <Content Remove="@(_ContentToUpdate)" />
       <Content Include="@(_ContentToUpdate)" ExcludeFromContentCheck="true" PublishFolderType="None" />
     </ItemGroup>
+
+  </Target>
+
+  <Target Name="_RemoveStaticWebAssetsDevelopmentManifest" BeforeTargets="GetCopyToOutputDirectoryItems">
+    <!-- Remove the development manifest from the output since its not used and it will otherwise get incorrectly loaded in production. -->
+    <ItemGroup>
+      <ContentWithTargetPath Remove="$(StaticWebAssetDevelopmentManifestPath)" />
```

Comment:
Ideally I'll update the Razor SDK to provide a flag that configures this behavior, this only happens because the appx seems to be grabbing all assets that are copied to the build output folder, which is not very standard in other .NET flavors which separate between build and publish (That's essentially what we leverage to disable the feature in prod, this file doesn't get copied to the publish output folder).

That said, this is the only file that static web assets puts in the build output for the app. Maui doesn't really use the development support for static web assets, since everything gets converted into MauiAssets, so I think adding a flag on the SDK that can turn this development behavior off and just leave the "copy everything to the wwwroot folder in 'publish'" is appropriate for these scenarios.